lib/appium_lib/ios/mobile_methods.rb

```diff
@@ -9,7 +9,8 @@ class << self
       #   ```
       def extended(_mod)
         Selenium::WebDriver::SearchContext.class_eval do
-          Selenium::WebDriver::SearchContext::FINDERS[:uiautomation] = '-ios uiautomation'
+          Selenium::WebDriver::SearchContext::FINDERS[:predicate] = '-ios predicate string'
```

Comment:
oh, I mistook.
Could you remain `Selenium::WebDriver::SearchContext::FINDERS[:uiautomation] = '-ios uiautomation'` here for accessibility id strategy in `XCUITest` case ? (we don't need switch I commented before, i think)